javascript/javascript1/week2/fullnameformal.js

```diff
@@ -0,0 +1,34 @@
+function getFullname (firstname, surname, useFormalName, gender){
+if (useFormalName) { 
+    if (gender==="man") {
+    console.log(`lord ${firstname} ${surname}`)   
+} else if(gender==="woman") {
+    console.log(`lady ${firstname} ${surname}`)
+}else{
+    console.log(`${firstname} ${surname}`)
+}   
+}
+}
+
+
+getFullname ('Benjamin', 'Hughes', true, "man");
```

Comment:
You should either choose to use `""` or `''`. I am personally a fan of using `""` with JS.